developer-docs-site/docs/guides/move-guides/move-on-aptos.md

```diff
@@ -48,24 +48,24 @@ With much more coming soon...
 
 ## Key Concepts in Move
 
-* Data should be stored within the account that owns it not the account that published the module.
-* Data flow should have minimal constraints with an emphasis on ecosystem usability
-* Prefer static type-safety over run-time safety via generics
-* A `signer` should be required to restrict access to adding or removing assets to an account unless it is explicitly clear
+* Data should be stored within the account that owns it, not the account that published the module.
+* Data flow should have minimal constraints with an emphasis on ecosystem usability.
+* Prefer static type-safety over run-time safety via generics.
+* A `signer` should be required to restrict access to adding or removing assets to an account unless it is explicitly clear.
 
 ### Data ownership
 
-Data should be stored within the account that owns it not the account that published the module.
+Data should be stored within the account that owns it, not the account that published the module.
 
 In Solidity, data is stored within the namespace of the account that created the contract. Typically this is represented by a map of an address to a value or of an instance id to the address of the owner.
 
 In Solana, data is stored within a distinct account associated with a contract.
 
-In Move, data can be stored within the module owner's account, but that creates the issue of ownership ambiguity and implies that two issues:
-1. It makes ownership ambiguous as the asset has no resource associated with the owner
-2. The module creator takes responsibility for the lifetime of that resources, e.g., rent, reclamation, etc
+In Move, data can be stored within the module owner's account, but that creates two issues:
+1. Ownership is ambiguous as the asset has no resource associated with the owner, and
```

Comment:
Here the original phrase, "It makes ownership ambiguous" is still correct, because this list is starting to enumerate what those two issues are. The "is" doesn't belong here.

Reply:
Here the correction was that both 1 and 2 must follow the same tonality. So if we go with `It makes ownership ambiguous`, then point 2 should be `It requires module creator to take lifetime responsibility`. Again – consider this just a suggestion.